build.ps1

```diff
@@ -112,5 +112,6 @@ if ($Test -and $PSCmdlet.ShouldProcess('Test assemblies', 'vstest.console.exe'))
     $TestAssemblies = Get-ChildItem -Recurse "$BinTestsFolder\*.Tests.dll"
     Write-Output "Testing..."
     # Add /Parallel switch when VS2015 Update 2 is more popular, as it's new in that version.
-    & $VSTestConsoleCommand.Path /TestAdapterPath:$BinTestsFolder $TestAssemblies
+    Set-Culture en-US
```

Comment:
Yes, I guess we can take it out now. I got 100% success on the tests on the last push. Will try running without this .